### ISSUE TITLE
new rule `font-face-max-count`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#963]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Add rule caring about count `font-face` keyword.

> Which rule, if any, is this issue related to?

e.g. `font-face-max-count`

> What CSS is needed to reproduce this issue?

```css
@font-face {
    font-family: Pompadur; /* Имя шрифта */
    src: url(fonts/pompadur.ttf); /* Путь к файлу со шрифтом */
}

@font-face {
    font-family: Pompadur; /* Имя шрифта */
    src: url(fonts/pompadur.ttf); /* Путь к файлу со шрифтом */
}

@font-face {
    font-family: Pompadur; /* Имя шрифта */
    src: url(fonts/pompadur.ttf); /* Путь к файлу со шрифтом */
}

@font-face {
    font-family: Pompadur; /* Имя шрифта */
    src: url(fonts/pompadur.ttf); /* Путь к файлу со шрифтом */
}

@font-face {
    font-family: Pompadur; /* Имя шрифта */
    src: url(fonts/pompadur.ttf); /* Путь к файлу со шрифтом */
}

```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "font-face-max-number": 5
  }
}
```

> Which version of stylelint are you using?

`5.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

no matter

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no matter

> What did you expect to happen?

No warnings

> What actually happened (e.g. what warnings or errors you are getting)?

Throw error if max count more than you

```shell
test.css
1:21    Expected 2 spaces (indentation)
```

**Comment by ntwb** [comment]
@evilebottnawi You listed the same CSS 5 times, were each of those meant to be different CSS examples?

**Comment by evilebottnawi** [comment]
@ntwb what?

**Comment by ntwb** [comment]
Nothing, ignore me, you closed this issue :)

**Comment by evilebottnawi** [comment]
@ntwb why? How about a discussion?

**Comment by evilebottnawi** [comment]
@ntwb sorry duplicate, bugs with github :)

**Comment by ntwb** [comment]
:)
